qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -72,14 +72,14 @@ These are the columns required for successfully submit your data to EBI:
 +----------------------------------+-------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
 | Field name                       | Format                  | Description                                                                                                                                         |
 +==================================+=========================+=====================================================================================================================================================+
-| ``collection_timestamp``         | ``mm/dd/yyyy hh:mm:ss`` | The time stamp (preferred) of when the sample was collected. Several format are accepted.                                                           |
-|                                  | or ``mm/dd/yyyy hh:mm`` |                                                                                                                                                     |
-|                                  | or ``mm/dd/yyyy hh``    |                                                                                                                                                     |
-|                                  | or ``mm/dd/yyyy``       |                                                                                                                                                     |
-|                                  | or ``mm/yyyy``          |                                                                                                                                                     |
+| ``collection_timestamp``         | ``yyyy-mm-dd hh:mm:ss`` | The time stamp (preferred) of when the sample was collected. Several format are accepted, all ISO 8601.                                             |
```

Comment:
User defined? We never got to discuss this in the meetings, do you, @josenavas have a preference?